auth-service/README.md

```diff
@@ -179,6 +179,36 @@ curl 'http://localhost:8088/auth-internal/bearer/user-info' \
 
 The response is similar to above, though the access token is not repeated.
 
+### Authenticating with the Delta API
+
+#### Obtaining an API token
+
+This endpoint requires authenticating with a valid AD username and password, and also a valid client id and secret combination. If running locally an id and secret must be added manually to the database. The test auth service DB contains `dev` and `dev` as an ID and secret pair. The stage auth service DB contains an id `dev-client` and a secret that can be found in AWS by the name `auth-service-dev-api-client-secret-staging`. When testing on test or stage, the domain should be replaced as appropriate. The origin domain should be `api.delta.[environment].communities.gov.uk`.
+
+```sh
+curl --request POST 'http://localhost:8088/delta-api/oauth/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--header 'Origin: http://localhost:8080' \
```

Comment:
Maybe be clear that the purpose of this Origin header is to test that swagger will work.